README.md

```diff
@@ -36,10 +36,11 @@ ParadeDB is currently in Public Beta. Star and watch this repository to get noti
   - [x] Dense vector search with [pgvector](https://github.com/pgvector/pgvector#pgvector)
   - [x] Hybrid search
 - [ ] Analytics
-  - [x] Real-time analytical queries and Delta Lake storage with [pg_analytics](https://github.com/paradedb/paradedb/tree/dev/pg_analytics#overview)
-  - [ ] High volume log ingest
-  - [ ] Kafka ingest
-  - [ ] Apache Iceberg storage
+  - [x] Acclerated analytical queries and column-oriented storage with [pg_analytics](https://github.com/paradedb/paradedb/tree/dev/pg_analytics#overview)
+  - [ ] External object store integrations (S3/Azure/GCS/HDFS)
```

Comment:
I would merge the two w/ Apache iceberg/etc cuz they go together, maybel ike


- [ ] Query external object stores (S3/Azure/GCS/HDFS) tables in Delta Lake/Iceberg format